source/_docs/pantheon-yml.md

```diff
@@ -36,6 +36,29 @@ protected_web_paths:
 * You may not be able to protect files or paths with special characters
 * Wait a few seconds for changes to take effect
 
+### HTTPS Redirects with HSTS Headers
+Ensure that your site will always use HTTPS to deliver content with `enforce_https`. Five values are available:
+
+- `off` (default: no https redirect)
+- `transitional` (Redirects to https and includes an HSTS header with a 5-minute duration)
+- `transitional+subdomains` (Redirects to https and includes an HSTS header with a 5-minute duration, including subdomains)
+- `full` (Redirects to https and includes an HSTS header with a 366-day duration)
+- `full+subdomains` (Redirects to https and includes an HSTS header with a 366-day duration, including subdomains)
+
+The preferred `hsts` configuration is shown below:
+```yaml
+enforce_https: full+subdomains
+```
+
+Note that any option with `+subdomains` should not be used if you have subdomains that are not hosted on Pantheon, unless you are sure they will behave correctly with these hsts settings. 
```

Comment:
We should be more specific about what it means to behave correctly with hsts settings. Their non-Pantheon site needs to be able to respond to https traffic for all requests.

Reply:
Ah, I see this is explained below, so "as explained below" might be a sufficient addition here.